Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/RomBannerRequestHandler.java

```diff
@@ -0,0 +1,33 @@
+package org.dolphinemu.dolphinemu.utils;
+
+import android.graphics.Bitmap;
+import android.util.*;
+
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.Request;
+import com.squareup.picasso.RequestHandler;
+
+import org.dolphinemu.dolphinemu.NativeLibrary;
+
+import java.io.IOException;
+import java.nio.IntBuffer;
+
+public class RomBannerRequestHandler extends RequestHandler {
```

Comment:
GameBannerRequestHandler? it doesn't use anything else other than loading the banner from the ISO file